src/view/widgets/accordion_widget.py

```diff
@@ -202,3 +224,13 @@ def get_config_value(self, config: dict, key: str, expected_type: type):
             return False
         else:
             return None
+
+    def on_accordion_card_change(self, event):
+        print(f"Accordion With Cards has changed!")
```

Comment:
logging would be more appropriate if this is relevant information, if these are for debugging purposes, please remove them

Reply:
Done.